domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandling.kt

```diff
@@ -482,6 +483,55 @@ data class Behandling internal constructor(
         }
     }
 
+    private inner class Underkjent : Tilstand {
+        override val status: BehandlingsStatus = BehandlingsStatus.UNDERKJENT
+
+        override fun sendTilAttestering(
+            saksbehandler: Saksbehandler
+        ): Either<AttestantOgSaksbehandlerKanIkkeVæreSammePerson, Behandling> {
+            if (erAttestantOgSakbehandlerSammePerson(saksbehandler)) {
+                return AttestantOgSaksbehandlerKanIkkeVæreSammePerson.left()
+            }
+            this@Behandling.saksbehandler = saksbehandler
+            if (erInnvilget()) {
```

Comment:
Metoden `erInnvilget()` alltid vil alltid gi false for `status = UNDERKJENT`. Dette fører til at alle behandlinger med `status = UNDERKJENT` vil ende opp som avslag dersom man sender dem til attestering!

Dette betyr ikke nødvendigvis at det er en logisk brist i `erInnvilget()` - her er det kanskje like naturlig at vi stiller spørsmålet om det i det hele tatt skal være mulig å sende noe direkte til attestering etter det har blitt sendt tilbake til saksbehandler? Ville vel forvente at årsaken til at det ble sendt tilbake var at noe måtte korrigeres, i så fall ville man kanskje tvinges via andre statuser slik at dette tilfellet ikke kunne oppstå?

Dersom det likevel skal være mulig å sende noe direkte til attestering igjen, må vi finne en løsning på problemet med `erInnvilget()`.

Reply:
Ja, dette var et av tingene jeg diskuterte litt med meg selv. Hva hvis attestant sender tilbake fordi noe han sjekket var feil. Men etter at saksbehandler nr 1 tok en runde 2, så viste det seg at attestanten tok feil? Da trengs det ikke nødvendigvis at saksbehandler gjør noen endringer, men bare sender direkte til attestanten igjen. 

Jeg vil mene at vi har begrunnelses feltet for denne grunn, men vi kan vel ikke være 100% sikker på at den kommer til å bli brukt for akkurat dette tilfellet for å oppdatere behandlingens status?